tests/Feature/ForumTest.php

```diff
@@ -59,6 +59,25 @@ public function the_thread_subject_cannot_be_an_url()
     }
 
     /** @test */
+    public function the_thread_subject_may_contain_multiple_spaces_between_words()
+    {
+        $tag = factory(Tag::class)->create(['name' => 'Foo Tag']);
+
+        $this->login();
+
+        $this->visit('/forum/create-thread')
+            ->submitForm('Create Thread', [
+                'subject' => 'This  is a Foo title',
+                'body' => 'This text explains how queues work in Laravel.',
+                'tags' => [$tag->id()],
+            ])
+            ->seePageIs('/forum/this-is-a-foo-title')
+            ->see('how queues work in Laravel.')
+            ->see('Foo Tag')
+            ->see('Thread successfully created!');
+    }
+
+    /** @test */
```

Comment:
You can remove this test again. The bug is covered with the smaller test for the validation rule above.

Reply:
Done!